app/models/organisation.rb

```diff
@@ -0,0 +1,24 @@
+class Organisation < ActiveRecord::Base
+  ORGANISATION_TYPES = %w{
+    webops
+    drs_call_center
+    laa_rota_team
+    court
+    custody_suite
+    law_firm
+    law_office
+  }
```

Comment:
Do we still need all these types? 
Prob only law_firm, law_office, court and custody_suite?